packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```diff
@@ -19,16 +19,16 @@
   &:indeterminate {
     + .label {
       .box {
-        border-color: var(--ColorBorderChecked);
-        background-color: var(--ColorBackgroundChecked);
+        border-color: var(--CheckedBoxBorderColor);
+        background-color: var(--CheckedBoxBackgroundColor);
```

Comment:
the state should occur after the component part so these would read:

```
var(--BoxCheckedBackgroundColor)
```

only variants precede the child component part name

similar feedback for other token names in this PR!